app/(main)/settings/team/agent-usage-table.tsx

```diff
@@ -0,0 +1,59 @@
+import { formatTimestamp } from "@/app/(playground)/p/[agentId]/canary/lib/utils";
```

Comment:
I used it because I thought it was better than defining a similar function, even though it is an import from outside the scope.